090-integrate-bridge-with-metrics-reporter.md

```diff
@@ -0,0 +1,117 @@
+# Integrate Bridge with Metrics Reporter
+
+In [SIP-064](https://github.com/strimzi/proposals/blob/main/064-prometheus-metrics-reporter.md) we introduced the [Strimzi Metrics Reporter](https://github.com/strimzi/metrics-reporter).
+This is a Kafka MetricsReporter plugin that directly exposes metrics in Prometheus format via an HTTP endpoint.
+
+## Current situation
+
+The HTTP Bridge allows users to enable or disable the metrics endpoint using the `KAFKA_BRIDGE_METRICS_ENABLED` environment variable.
+When this variable is set to true, the Bridge creates the JMX Exporter's JmxCollector class with hard coded configuration.
+This configuration is a JMX metrics exporter YAML distributed as an embedded resource within the Bridge's JAR file.
+
+When deploying the Bridge through the Cluster Operator, a similar configuration is available in the KafkaBridge's CRD:
+
+```sh
+spec:
+  enableMetrics: true
+```
+
+This is different from how the metrics endpoint is enabled in all the other major components deployed by the Cluster Operator.
+The Cluster Operator exposes metrics through the [Prometheus JMX Exporter](https://github.com/prometheus/jmx_exporter), which can be configured using the shared `metricsConfig` schema.
+This schema has a type property that only allows the `jmxPrometheusExporter` value, and a reference to a ConfigMap containing its configuration:
+
+```sh
+metricsConfig:
+  type: jmxPrometheusExporter
+  valueFrom:
+    configMapKeyRef:
+      name: $COMPONENT_NAME-metrics
```

Comment:
maybe a note to highlight what `$COMPONENT_NAME` means here and possible values?

Reply:
Yes, this is confusing. Let me fix it.